robosuite/scripts/render_dataset_with_omniverse.py

```diff
@@ -74,6 +74,23 @@
     "--keep_models", type=str, nargs="+", default=[], help="(optional) keep the model from the Mujoco XML file"
 )
 
+# adding rendering types
```

Comment:
Thanks! Could we also include instructions for how to install isaac successfully e.g. in this file?